regulatory_compliance/postgres.sp

```diff
@@ -406,7 +406,7 @@ query "postgres_db_server_allow_access_to_azure_services_disabled" {
         azure_postgresql_server,
         jsonb_array_elements(firewall_rules) as r
       where
-        r -> 'FirewallRuleProperties' ->> 'endIpAddress' = '255.255.255.255'
+        r -> 'FirewallRuleProperties' ->> 'endIpAddress' = '0.0.0.0'
```

Comment:
This was updated as per the CIS document,but I think we should use `or` clause instead of `and`.
<img width="745" alt="image" src="https://github.com/turbot/steampipe-mod-azure-compliance/assets/46913995/cffc316b-df3e-439f-a85e-c3d766922eb2">